.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release
+
+on:
+  release:
+    types: [released, prereleased]
+
+env:
+  CI: true
+  NODE_OPTIONS: --max_old_space_size=4096
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_USER: postgresql
+          POSTGRES_DB: users
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+      mongodb:
+        image: mongo:4.2.9
```

Comment:
Could we test multiple versions with a Matrix?

Reply:
I think github actions have fair usage so we need to be careful to not hit the quota ;D

Reply:
Same reason as https://github.com/aerogear/graphback/pull/1938#discussion_r476282885. But testing with at least two major release makes a lot of sense.

Reply:
Okay, we could settle on only having it on release action, we would unlikely hit the quota then

Reply:
:+1: Good point.